### DISCUSSION TITLE
Allow parameter to be supplied to CIGALE as a text file

### DISCUSSION THREAD

**Discussion by Bovornpratch** [mboquien/cigale#28] [Ideas]
Apologies if I misunderstand CIGALES capabilities. I was wondering if it is possible for CIGALE to fix some  parameters for each object in the pdf_analysis mode.

At first I though it would possible to do this by supplying a parameter file but the info said that using it with the pdf_analysis module will yield incorrect results.

This request is driven by the recent advancements in the wide frequency radio interferometry instruments which can measure the the radio slope within the band. This means that some radio datasets would be able to supply the radio flux in for example L-band with a radio slope (see https://arxiv.org/abs/2006.08551 for examples) for several thousand sources. In this case, radio.radio_alpha can be fixed if there is only one single data point. 

I was thinking this may be an interesting feature if not yet implemented.

**Comment by mboquien** [comment]
For this the most simple way is to add a column in the input flux file with the name of the parameter and then add it to the list of parameters to be fitted in `pcigale.ini` (this is right below `bands`). Depending on the size of the uncertainty (just the same column ending with `_err`), CIGALE will preferentially select models with the value given in the input file.
